app/models/metasploit_data_models/automatic_exploitation/match_result.rb

```diff
@@ -0,0 +1,36 @@
+class MetasploitDataModels::AutomaticExploitation::MatchResult < ActiveRecord::Base
+  attr_accessible :match_id, :run_id, :state
+
+  # Match results are in one of two states:
+  #  - "succeeded" (exploit gained access)
+  #  - "failed" (exploit did NOT gain access)
+  VALID_STATES = ["succeeded", "failed"]
+
+  #
+  # ASSOCIATIONS
+  #
+
+  belongs_to :match,
+             class_name: 'MetasploitDataModels::AutomaticExploitation::Match',
+             dependent: :destroy
+
+  belongs_to :run,
+             class_name: 'MetasploitDataModels::AutomaticExploitation::Run'
+
+  #
+  # VALIDATIONS
+  #
+
+  # must be present and one of allowable values
+  validates :state,
+            presence: true,
+            inclusion: VALID_STATES
```

Comment:
I thought you had to do this as

```ruby
inclusion: {
  in: VALID_STATES
}
```

Are you sure this works as expected.  There are no specs for this class.